service/src/test/java/bio/terra/cbas/controllers/TestRunsApiController.java

```diff
@@ -77,7 +77,7 @@ class TestRunsApiController {
   @Autowired private ObjectMapper objectMapper;
   @MockBean private SamService samService;
 
-  @MockBean private MeterRegistry meterRegistry;
+  @MockBean private MicrometerMetrics micrometerMetrics;
```

Comment:
Similarly here to my comment on the prior commit, I think if you use `@Autowired`, the framework should inject a `MicrometerMetrics` instance, which itself will be autowired (to include a `SimpleMetricRegistry`).  Then you wouldn't need to mock out this bean either (although I guess you can, if that's your intent).